src/components/ReviewList.jsx

```diff
@@ -1,21 +1,31 @@
-import { useEffect, useState } from "react"
+import { useEffect, useState, useContext } from "react"
 import ReviewCard from "./ReviewCard";
 
-export default function ReviewList () {
+import { CategoryContext } from "../contexts/CategoryContext";
 
+export default function ReviewList () {
+    
 const axios = require("axios");
 const [reviews, setReviews] = useState([]);
+const {selectedCategory} = useContext(CategoryContext)
+
+let categoryTitleName = selectedCategory.split("-").join(" ")
+categoryTitleName = categoryTitleName[0].toUpperCase() + categoryTitleName.substring(1);
 
 useEffect(() => {
     axios.get("https://nc-games-jk.herokuapp.com/api/reviews").then(({data}) => {
-        setReviews(data.reviews);
+        const newReviews = data.reviews.filter((review) => {
```

Comment:
We highly don't recommend fetching all the reviews and then filtering them. You have an endpoint in your API that allows to query reviews by category, so make use of that :)